hack/compare-gen-manifests.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Does it make sense to add makefile rules to invoke this?

Reply:
good idea added